src/tree-manager/index.js

```diff
@@ -229,7 +249,7 @@ class TreeManager {
       if (node.checked) {
         tags.push(node)
 
-        if (!this.hierarchical) {
+        if (!this.hierarchical || this.radioSelect) {
```

Comment:
For radio, you're already keeping track of selected, so it'll be faster if we just return that instead of traversing the tree. Same for simple-select but simple select being a shorter list, the performance impact of traversing was negligible/minimal.

Reply:
Done, returns tag for currentChecked